lib/solargraph.rb

```diff
@@ -56,8 +56,6 @@ def self.logger
 
   # A helper method that runs Bundler.with_unbundled_env or falls back to
   # Bundler.with_clean_env for earlier versions of Bundler.
-  #
-  # @return [void]
```

Comment:
We use this in a place where we save the return value.  The return type and the yieldreturn here really should be `param<T>` once we support that.